tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -597,6 +584,18 @@ def test_model_monitoring_voting_ensemble(self):
         # invoke the model before running the model monitoring batch job
         iris_data = iris["data"].tolist()
 
+        # Simulating invalid requests
+        invalid_input = ["n", "s", "o", "-"]
+        for _ in range(10):
+            try:
+                serving_fn.invoke(
+                    "v2/models/VotingEnsemble/infer",
+                    json.dumps({"inputs": [invalid_input]}),
+                )
+            except RuntimeError:
+                pass
```

Comment:
Is it on purpose that we accept it both raising an error and not raising an error?

Reply:
So it should always return an error? If so, better use `pytest.raises` so it will fail if it doesn't raise the error.

Reply:
replaced with `pytest.raises`.